src/DynamoCore/Linting/LinterManager.cs

```diff
@@ -20,8 +20,10 @@ namespace Dynamo.Linting
     public class LinterManager : NotificationObject, IDisposable
     {
         #region Private fields
+        private const string NONE_DESCRIPTOR_GUID = "7b75fb44-43fd-4631-a878-29f4d5d8399a";
         private readonly IExtensionManager extensionManager;
         private LinterExtensionDescriptor activeLinter;
+        internal LinterExtensionDescriptor DefaultDescriptor => new LinterExtensionDescriptor(NONE_DESCRIPTOR_GUID, Properties.Resources.NoneLinterDescriptorName);
```

Comment:
Maybe definition of default descriptor should be in LinterExtensionDescriptor class as a static ?